README.md

```diff
@@ -1 +1,278 @@
-# goat-guardian
+# Goat Guardian
+
+[![Build Status](https://secure.travis-ci.org/arow-oss/goat-guardian.svg)](http://travis-ci.org/arow-oss/goat-guardian)
+[![Hackage](https://img.shields.io/hackage/v/goat-guardian.svg)](https://hackage.haskell.org/package/goat-guardian)
+[![Stackage LTS](http://stackage.org/package/goat-guardian/badge/lts)](http://stackage.org/lts/package/goat-guardian)
+[![Stackage Nightly](http://stackage.org/package/goat-guardian/badge/nightly)](http://stackage.org/nightly/package/goat-guardian)
+
+Goat Guardian is a reverse-proxy authentication server.  You can run it in
+front of an existing web application.  It will handle all authentication,
+including OAuth and email-based login.
+
+Goat guardian is meant to make it easy to add support for authentication to
+your web application.  It works great for demos, prototypes, internal web
+applications, rapid development with a small team.  It even works well for
+larger applications and teams that want an easy way to implement multiple forms
+of authentication!
+
+## Introduction
+
+Goat Guardian is a reverse-proxy that runs in front of your web application.
+It proxies requests to your web application.  It is similar to Apache or nginx.
+
+Here is a simple network diagram of what it looks like to use Goat Guardian:
+
+![Goat Guardian Simple Network Diagram](./img/goat-guardian-network-no-db.png)
+
+Goat Guardian received requests from the end-user.  If it is a request to
+authenticate, Goat Guardian will directly handle it.  If it is any other
+request, it will be forwarded to your web application.
+
+## Language
```

Comment:
At first, I could not realize that this section is for defining verbs used in this document.
How about using more helpful title such as "Word definition"?

Reply:
In commit 6c1887174 I changed this to "Terminology used in this document".  I think this is more natural in English.

Is this understandable?

Reply:
Nice!